exposed-jodatime/src/test/kotlin/org/jetbrains/exposed/JodaTimeTests.kt

```diff
@@ -33,4 +70,10 @@ fun equalDateTime(d1: DateTime?, d2: DateTime?) = try {
     false
 }
 
-val today: DateTime = DateTime.now().withTimeAtStartOfDay()
\ No newline at end of file
+val today: DateTime = DateTime.now().withTimeAtStartOfDay()
+
+object Cities : Table() {
```

Comment:
Please rename to something like `CitiesTime` as there are another `Cities` in tests that could confuse when you use import.